src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,31 @@
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 import './ChatEntry.css';
 
-const ChatEntry = () => {
+const ChatEntry = ({id, sender, body, timeStamp, liked, onToggleLike}) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className= {`chat-entry ${liked ? 'liked' :'local'}`}>
```

Comment:
Not sure if this ternary is a mistake or not, seems like the `liked` part is messing up some of the CSS?